src/coffeeandtacos/TacoParser.java

```diff
@@ -8,8 +8,38 @@
     public Trackable Parse(String line) {
         
         logger.logInfo("Begin Parsing");
+        if(line == null)
+            {
+                logger.logWarning("Line was null");
+                return null;
+            }
+        // Take your line and use line.Split(',') to split it up into an array of strings, separated by the char ','
+        var cells = line.split(",");
         
         // Do not fail if one record parsing fails, return null
-        return null; //TODO Implement
+        // If your array.Length is less than 3, something went wrong
+        if (cells.length < 3 || cells.length > 3)
+        {
+            // Log that and return null
+            logger.logWarning("The length was the array less than 3");
```

Comment:
This will log the error "The length was the array less than 3", even if it is greater than 3.